### PR TITLE
best practices: fine-grained cpu orchestration

### PR DESCRIPTION
**Pull request by ZiMengSheng**

### CODE REVIEW

File: docs/best-practices/fine-grained-cpu-orchestration.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
+# Coordinated sharing of CPU resources in Colocation Scenarios - Fine-grained CPU Orchestration
```
Comment (eahydra) [inline docs/best-practices/fine-grained-cpu-orchestration.md:1]: The title is too long.. The title of PR looks better and can be used.

File: docs/best-practices/fine-grained-cpu-orchestration.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+## Introduction
+
+In a cloud-native environment, cluster providers often deploy different types of workloads in the same cluster, leveraging different peak effects of different services to achieve time-sharing multiplexing of resources and avoid resource waste. However, colocation of different types of workloads often leads to resource competition and mutual interference. The most typical scenario is the colocation  of online and offline workloads. When more computing resources are occupied by offline workloads, the response time of online loads will be affected; when more computing resources are occupied by online workloads for a long time, the task completion time of offline workloads cannot be guaranteed. This phenomenon belongs to the Noisy Neighbor problem.
```
Comment (eahydra) [inline docs/best-practices/fine-grained-cpu-orchestration.md:5]: Recommend: `There are many different types of workload in Kubernetes cluster, and.........`

**Comment by hormes** [comment]
You need to add a link [here](https://github.com/koordinator-sh/website/blob/main/sidebars.js).

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**